frontend/src/components/form/SearchUsers.vue

```diff
@@ -53,18 +53,17 @@ const onCancel = () => {
 };
 
 const onChange = (event: any) => {
-  // Duplicate user check
-  if( !props.permissions.some(perm => perm.userId === event.value?.userId) ) {
+  if(isProxy(event.value)) {
+    // Duplicate user check
+    if( !props.permissions.some(perm => perm.userId === event.value?.userId) ) {
     // Set state
-    selectedUser.value = event.value;
-  }
-  else {
+      selectedUser.value = event.value;
+    }
+    else {
     // Invalid
-    selectedUserIsInvalid.value = true;
+      selectedUserIsInvalid.value = true;
+    }
```

Comment:
We should be careful about how we're naming our variables especially if they have the potential to be negative affirmations. In this case, I can see why it was named invalid (p-invalid class), but I would consider renaming this to something more assertive like `invalidSelectedUser` or `isInvalidSelectedUser` - by declaring invalid earlier it makes it more salient that we're focusing on the negative case instead of emphasizing on the user.